Documentation/subcommands/run.md

```diff
@@ -84,16 +84,79 @@ rkt: warning: signature verification has been disabled
 
 ## Mount Volumes into a Pod
 
-Volumes are defined in each ACI and are referenced by name. Volumes can be exposed from the host into the pod (`host`) or initialized as empty storage to be accessed locally within the pod (`empty` pending [rkt #378][rkt #378]). Each volume can be selectively mounted into each application at differing mount points or not mounted into specific apps at all.
+Each ACI can define a [list of mount points](https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema) that the app is expecting external data to be mounted into:
 
-[rkt #378]: https://github.com/coreos/rkt/issues/378
+```json
+{
+    "acKind": "ImageManifest",
+    "name": "example.com/app1",
+    ...
+    "app": {
+        ...
+        "mountPoints": [
+            {
+                "name": "data",
+                "path": "/var/data",
+                "readOnly": false
+            }
+        ]
+    }
+    ...
+}
+```
+
+To fulfill these mount points, volumes are used. There are today two kinds of volumes:
+- `host` volumes that can expose a directory or a file from the host to the pod
+- `empty` volumes that initialize an empty storage to be accessed locally within the pod.
+
+Each volume can be selectively mounted into each application at differing mount points. For example, Any volumes that are specified but do not have a matching mount point will be silently ignored.
+
+### Mounting Volumes
+
+For `host` volumes, the `--volume` flag allows you to specify each mount, its type, the location on the host, and whether the volume is read-only or not:
+
+```
+# rkt run --volume data,kind=host,source=/srv/data,readOnly=false app1 --- app2
+```
+
+If you don’t intend to persist the data and you just want to have a volume shared between all the apps in the pod, you can use an `empty` volume:
+
+```
+# rkt run --volume data,kind=empty,readOnly=false app1 --- app2
+```
+
+The volume is then mounted into each app running to the pod based on information defined in the ACI manifest.
+
+### Mounting Volumes without Mount Points
+
+If the ACI doesn’t have any mount points defined in its manifest, you can still mount volumes using the `--mount` or `--inject-volume` flags.
+
+With `--mount` you define a mapping between volumes and a path in the app. This will override any mount points in the image manifest.
+In the following example, the `--mount` option is positioned after the app name; it defines the mount only in that app:
+
+```
+# rkt run --volume data,kind=host,source=/srv/data \
```

Comment:
I'll change the volume name and targets.

I don't really want to focus on the mountpoint overriding case, I think it is a pretty weird use-case and rkt will do what the user expects.